samples/kotlin-mcp-client/src/main/kotlin/io/modelcontextprotocol/sample/client/MCPClient.kt

```diff
@@ -50,19 +62,19 @@ class MCPClient : AutoCloseable {
                 add(serverScriptPath)
             }
 
-            // Start the server process
+            // Start server process
             val process = ProcessBuilder(command).start()
 
-            // Setup I/O transport using the process streams
+            // Setup transport
             val transport = StdioClientTransport(
                 input = process.inputStream.asSource().buffered(),
                 output = process.outputStream.asSink().buffered()
             )
 
-            // Connect the MCP client to the server using the transport
+            // Connect MCP client
             mcp.connect(transport)
 
-            // Request the list of available tools from the server
+            // List tools
```

Comment:
In my opinion, it’s better to keep detailed comments in the example. For new users, a comment like `List tools` doesn’t provide any additional information